### ISSUE TITLE
xrdp 0.9.0+ IdleTimeLimit ?

### ISSUE THREAD

**Issue by cpcn** [neutrinolabs/xrdp#1212]
I have several Debian 9 instances with xrdp 0.9.6 installed (from Debian repo) Problem is the rdesktop / xrdp connection is closed and killed on "inactivity"

I have older units (xrdp 0.9.0 manually built from source) and the sesman.ini has 

```
IdleTimeLimit - when to disconnect idle sessions
Type: integer
Default: 0
if not zero, the seconds without mouse or keyboard input before disconnect
not complete yet
IdleTimeLimit=0
```

This option is not available with later xrdp releases?
If not, any suggestions on how to maintain the connections?

**Comment by metalefty** [comment]
See #652.

**Comment by metalefty** [comment]
I'm working on this.  Will be implemented next release.

**Comment by cpcn** [comment]
K Great! it's a good feature. 
Did alot of looking around ..I think my problem was due to the screensaver or power management energy star equipment.

**Comment by metalefty** [comment]
Solved by #1227.
